capstone/capdb/models.py

```diff
@@ -259,56 +260,65 @@ def __str__(self):
 
     def update_case_metadata(self):
         data = get_case_metadata(self.orig_xml)
-        citation, created = Citation.objects.get_or_create(
-            cite=data["citation"],
-            type=data["citation_type"])
+        volume_metadata = VolumeMetadata.objects.get(barcode=data["volume_barcode"])
+        reporter = Reporter.objects.get(id=volume_metadata.reporter_id)
```

Comment:
Here it's a little clearer to use model attributes (which should run similar queries behind the scenes):

```
volume_metadata = self.volume.volume_metadata
reporter = volume_metadata.reporter
```